// Validating benchmarks:
// ***** BenchmarkRunner: Start   *****
// ***** Found 2 benchmark(s) in total *****
// ***** Building 1 exe(s) in Parallel: Start   *****
// start dotnet restore  /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc
// command took 2.37s and exited with 0
// start dotnet build -c Release  --no-restore /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc
// command took 3.72s and exited with 0
// ***** Done, took 00:00:06 (6.24 sec)   *****
// Found 2 benchmarks:
//   BenchMarkService.LoadDataTask: DefaultJob
//   BenchMarkService.LoadDataValueTask: DefaultJob

// **************************
// Benchmark: BenchMarkService.LoadDataTask: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "e5bf2af3-99ee-4402-96fc-3a940f3aeadc.dll" --benchmarkName "BenchmarkExample.BenchMarkService.LoadDataTask" --job "Default" --benchmarkId 0 in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc\bin\Release\net5.0
// BeforeAnythingElse

// Benchmark Process Environment Information:
// Runtime=.NET 5.0.13 (5.0.1321.56516), X64 RyuJIT
// GC=Concurrent Workstation
// Job: DefaultJob

OverheadJitting  1: 1 op, 238400.00 ns, 238.4000 us/op

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at BenchmarkExample.BenchMarkService.LoadDataTask() in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\Program.cs:line 41
   at BenchmarkDotNet.Autogenerated.Runnable_0.<.ctor>b__3_4() in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc\e5bf2af3-99ee-4402-96fc-3a940f3aeadc.notcs:line 189
   at BenchmarkDotNet.Autogenerated.Runnable_0.WorkloadActionNoUnroll(Int64 invokeCount) in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc\e5bf2af3-99ee-4402-96fc-3a940f3aeadc.notcs:line 842
   at BenchmarkDotNet.Engines.Engine.RunIteration(IterationData data)
   at BenchmarkDotNet.Engines.EngineFactory.Jit(Engine engine, Int32 jitIndex, Int32 invokeCount, Int32 unrollFactor)
   at BenchmarkDotNet.Engines.EngineFactory.CreateReadyToRun(EngineParameters engineParameters)
   at BenchmarkDotNet.Autogenerated.Runnable_0.Run(IHost host, String benchmarkName) in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc\e5bf2af3-99ee-4402-96fc-3a940f3aeadc.notcs:line 168
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in E:\localcode\CodeReview\C#\dotnet\net5.0\BenchmarkExample\BenchmarkExample\bin\Release\net5.0\e5bf2af3-99ee-4402-96fc-3a940f3aeadc\e5bf2af3-99ee-4402-96fc-3a940f3aeadc.notcs:line 52
// AfterAll
// Benchmark Process 35812 has exited with code -1.
// * Artifacts cleanup *
